jeeves/report.py

```diff
@@ -163,6 +170,7 @@ def run_report(config, blockers, preamble_file, server, header, test_email, no_e
 	summary['total_success'] = "Total SUCCESS:  {}/{} = {}%".format(num_success, num_jobs, percent(num_success, num_jobs))
 	summary['total_unstable'] = "Total UNSTABLE: {}/{} = {}%".format(num_unstable, num_jobs, percent(num_unstable, num_jobs))
 	summary['total_failure'] = "Total FAILURE:  {}/{} = {}%".format(num_failure, num_jobs, percent(num_failure, num_jobs))
+	summary['total_coverage'] = "Total FAILURE coverage:  {}/{} = {}%".format(num_covered, num_failure, percent(num_covered, num_failure))
```

Comment:
Again why only FAILURE jobs? Wouldn't it be better to show overall coverage of non-passing jobs?

Reply:
+1

Reply:
I was thinking about removing failure constraint and making it more generic. How about total coverage(including aborted,failures and unstable)? Does it works for you?

Reply:
Agreed; I would say let's track the metric for all jobs that don't have "SUCCESS" status, i.e. "UNSTABLE", "FAILURE", "ABORTED", "NO_KNOWN_BUILDS"